packages/blockchain-wallet-v4-frontend/src/scenes/app.tsx

```diff
@@ -95,6 +98,25 @@ const App = ({
   walletDebitCardEnabled
 }: Props) => {
   const Loading = isAuthenticated ? WalletLoading : AuthLoading
+
+  useEffect(() => {
+    const checkAuth = async () => {
+      if (isPlugin()) {
+        const isAuthenticated = await isSessionActive()
+        if (
+          !isAuthenticated &&
+          // @ts-ignore
+          history.location.pathname !== '/login' &&
+          // @ts-ignore
+          history.location.pathname !== '/signup'
+        ) {
+          await chrome.tabs.create({ url: chrome.runtime.getURL('index-tab.html') })
+        }
```

Comment:
we should add `window.close()` after `await chrome.tabs.create({ url: chrome.runtime.getURL('index-tab.html') })`